clearly-defined/src/main/kotlin/Extensions.kt

```diff
@@ -17,12 +17,34 @@
  * License-Filename: LICENSE
  */
 
+@file:Suppress("MatchingDeclarationName")
```

Comment:
Can you add a sentence to the commit message explaining how the harvest status is considered? E.g. only upload if the component is already harvested?

Reply:
Done.